CMakeLists.txt

```diff
@@ -1,13 +1,15 @@
 cmake_minimum_required(VERSION 3.0.0)
-project("Lily")
+project("Lily" C)
 
 if(MSVC)
     add_definitions(/W4)
     add_definitions(/wd4201)
     add_definitions(/wd4204)
     add_definitions(/wd4214)
     add_definitions(/MP)
+    set(CMAKE_CTEST_COMMAND	copy lily.exe ..)
 else()
+    set(CMAKE_CTEST_COMMAND	cp lily ..)
```

Comment:
My lily dir is also called lily. This is to chdir from the cmake build dir down to the root dir. Same as in the appveyor or travis recipes.